docs/pages/guides/tutorials/hello-kubernetes-with-dapr.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: Hello Kubernetes with Dapr
+abstract: |
+  Dive into this step-by-step tutorial on deploying Dapr in Kubernetes using Meshery. Learn to deploy a Python app for generating messages and a Node app for consuming and persisting them, all orchestrated with Dapr for enhanced application integration.
+permalink: guides/tutorials/hello-kubernetes-with-dapr
+redirect_from: guides/tutorials/hello-kubernetes-with-dapr
+model: kubernetes
+type: guides
+category: tutorials
+language: en
+list: include
+---
+
+
+### Introduction:
+This tutorial demonstrates how to deploy Dapr in a Kubernetes cluster using Meshery. You'll deploy a Python app for message generation and a Node app for consumption and persistence following the classic Hello World example with Dapr. Let's dive in and explore the architecture diagram that illustrates this setup.
+![](./screenshots/Architecture_Diag.png)
+
+### Prerequisites:
+- Basic understanding of containerization, Kubernetes concepts.
+- Access to the Meshery Playground. If you don't have an account, sign up at [Meshery Playground](https://playground.meshery.io).
+
+### Lab Scenario:
+You'll be deploying `Hello Kuberentes` applications with the help of Meshery, which has a Python app that generates messages and a Node app that consumes and persists them.
+
+### Learning Objective:
+Upon completing this tutorial, you will:
+- Deploy Hello Kubernetes with Dapr using Meshery.
+- Implement a Python app for message generation and a Node app for consumption and persistence with Dapr.
+- Explore Dapr's features for service invocation, state management, and application observability in Kubernetes.
+
+### Steps:
+
+#### Access Meshery Playground
+- Using your credentials, log in to the [Meshery Playground](https://playground.meshery.io). You should be at the dashboard upon successful login.
+
+#### Step 1: Install Dapr on your meshery playground. 
```

Comment:
It would be nice to be consistent throughout the documentation, such as using capitalization like `Meshery`.